README.md

```diff
@@ -358,7 +358,7 @@ Many people all over the world have helped make this project better.  You'll wan
 
 ### Community
 
-* Chat with other community members [![Join the chat at https://gitter.im/azure/azure-sdk-for-c](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/azure/azure-sdk-for-c?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+* Chat with other community members [![Join the chat at https://gitter.im/azure/azure-sdk-for-c](https://badges.gitter.im/Join%20Chat.png)](https://gitter.im/azure/azure-sdk-for-c?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)

```

Comment:
The text seem to get blurry with the png :(
![image](https://user-images.githubusercontent.com/6527137/81639026-9bd28580-93cf-11ea-8e37-0e784cfea1d7.png)

Compared to the svg:
![image](https://user-images.githubusercontent.com/6527137/81639046-a725b100-93cf-11ea-84a8-8e92acbf505f.png)

Is there another way to get svg to work (or maybe some other image format that renders better)? Can we manually include it, using `IMAGE_PATH` and the `\image` command?
https://stackoverflow.com/questions/29929303/svg-image-files-in-doxygen

Reply:
Looks like updating the local version of doxygen fixes the problem.